### ISSUE TITLE
Loading model fails ('EPDTC' object has no attribute '_ep_approximation')

### ISSUE THREAD

**Issue by otte-c** [SheffieldML/GPy#380]
Hello,

Loading a 'pickled' classification model throws an exception. Can be reproduced by the following code. Tested with GPy-devel 1.0.3 but the latest version also seems to be affected.
Any advice or workaround available?

Thanks and regards
Clemens

```
import GPy
import numpy as np
X = np.random.rand(4,2)
y = np.array([0, 0, 1, 1]).reshape(4,1) 
m = GPy.models.SparseGPClassification(X, y)
m.pickle('test.pkl')
m = GPy.load('test.pkl')  # AttributeError: 'EPDTC' object has no attribute '_ep_approximation'
```

**Comment by mzwiessele** [comment]
Good spotting, this is a bug and it is fixed in [devel e4d76a1]

**Comment by debipe20** [comment]
Hello,
I have installed GPy but while running code by importing GPy it is showing "ModuleNotFoundError: No module named 'GPy'."
